oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/AuthorizationServerRuntimeHintsRegistrar.java

```diff
@@ -152,21 +142,15 @@ public void registerHints(RuntimeHints hints, ClassLoader classLoader) {
 					Class.forName("org.springframework.security.oauth2.client.jackson2.OidcIdTokenMixin"));
 			this.reflectionHintsRegistrar.registerReflectionHints(hints.reflection(),
 					Class.forName("org.springframework.security.oauth2.client.jackson2.OidcUserInfoMixin"));
-			this.reflectionHintsRegistrar.registerReflectionHints(hints.reflection(),
-					Class.forName("org.springframework.security.oauth2.server.authorization.jackson2.OAuth2AuthorizationRequestMixin"));
-			this.reflectionHintsRegistrar.registerReflectionHints(hints.reflection(),
-					Class.forName("org.springframework.security.oauth2.server.authorization.jackson2.OAuth2TokenFormatMixin"));
-		} catch (ClassNotFoundException ex) {
+			this.reflectionHintsRegistrar.registerReflectionHints(hints.reflection(), Class.forName(
```

Comment:
could we also add 
`this.reflectionHintsRegistrar.registerReflectionHints(hints.reflection(),
          Class.forName("org.springframework.security.web.jackson2.DefaultSavedRequestMixin"));`
here